app/views/layout.mote

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+    <head></head>
+    <body>
+
+
+
+        {{content}}
```

Comment:
none of this loads up so I can't mark any interactions. The content isn't there and it just shows the dev tools links.